easybuild/easyblocks/t/tensorflow.py

```diff
@@ -35,14 +35,13 @@
 import re
 import stat
 import tempfile
-from distutils.version import LooseVersion
 
 import easybuild.tools.environment as env
 import easybuild.tools.toolchain as toolchain
 from easybuild.easyblocks.generic.pythonpackage import PythonPackage, det_python_version
 from easybuild.easyblocks.python import EXTS_FILTER_PYTHON_PACKAGES
 from easybuild.framework.easyconfig import CUSTOM
-from easybuild.tools import run
+from easybuild.tools import run, LooseVersion
```

Comment:
I would prefer retaining compatibility with older EasyBuild versions, to cater to people who pull in this updated easyblock via `--include-easyblocks-from-pr`, yet are still using an EasyBuild version < 4.7.0:

```python
# try to use LooseVersion provided by EasyBuild v4.7.0+,
# fall back to distutils.version.LooseVersion (which works for Python < 3.12)
try:
    from easybuild.tools import LooseVersion
except ImportError:
    from distutils.version import LooseVersion

Reply:
This is 6 lines instead of a single word which IMO isn't worth the effort. Especially as this is EasyBuild which is very easy (haha!) to install and update I don't want to introduce workarounds for people insisting to use old versions with new EasyBlocks. We don't even know if that works at all as other things in framework might have changed so I'd really reduce the usecase of `--include-easyblocks/easyconfigs` to those using the develop branch

Reply:
Yeah, fair point, agreed, let's leave this as is...